server/app/app.py

```diff
@@ -53,5 +40,33 @@
 app.add_url_rule("/", endpoint="index")
 
 
+def run(args):
+    global data
```

Comment:
generally not a fan of globals.

consider attaching the data object to the app object (possibly in a separate PR).

consider also instantiating the app object here (i.e., in `run()`), and converting any references to `server.app.app.app` to [flask.current_app](http://flask.pocoo.org/docs/1.0/api/#flask.current_app) in a separate PR.

Reply:
I didn't like it either, I prefer your idea. I wasn't quite sure what the best alternative to be and wanted to keep this PR relatively small. I like your idea, let's discuss later today.